components/src/dynamo/vllm/multimodal_handlers/multimodal_pd_worker_handler.py

```
@@ -0,0 +1,416 @@
+
+        gen = self.engine_client.generate(
+            prompt=TokensPrompt(
+                prompt_token_ids=prefill_only_request.engine_prompt["prompt_token_ids"],
+                multi_modal_data=multi_modal_data,
+            ),
+            sampling_params=prefill_only_request.sampling_params,
+            request_id=prefill_only_request.request_id,
+        )
+
+        # Drain prefill generator (max_tokens=1, expect a single response)
+        async for prefill_response in gen:
```

Ruff (0.15.0)

[warning] Loop control variable `prefill_response` not used within loop body

(B007)